jsontime/integer.go

```diff
@@ -40,6 +47,32 @@ func (um *UnixMilli) UnmarshalJSON(data []byte) error {
 	return parseTime(data, time.UnixMilli, &um.Time)
 }
 
+func (um UnixMilli) Value() (driver.Value, error) {
+	return time.Time(um.Time).UnixMilli(), nil
```

Comment:
```suggestion
	return um.UnixMilli(), nil
```

(same for the rest of the methods)